patch/msft-kernel-ipv6-ra-metric-sysctl.patch

```diff
@@ -0,0 +1,194 @@
+commit 7c12a9ecf4e0c782875e399ef43220a7fce2f8ca
+Author: pchaudha <pchaudha@contoso.com>
+Date:   Tue Dec 29 05:05:01 2020 +0000
+
+    ip6 ra sysctl
```

Comment:
It would be great, if you elaborated here.

Reply:
Hope now description is better.